packages/babel-generator/test/fixtures/comments/simple-a-lot-of-line-comment/expected.js

```diff
@@ -20,6 +20,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 // THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+
```

Comment:
I this case I think it's good because it's actually closer to the `actual.js` whitespacing: https://github.com/loganfsmyth/babel/blob/9ceb43c95f27b582b1a1163c7e64e416b3c32c94/packages/babel-generator/test/fixtures/comments/simple-a-lot-of-line-comment/actual.js